prow/config.yaml

```diff
@@ -301,6 +301,29 @@ branch-protection:
               - "run-e2e-tests / run-e2e-tests"
               - "markdown-link-check"
               - "validate-labels"
+        lifecycle-manager:
+          required_status_checks:
+            contexts:
+              - "E2E (kyma-deletion)"
+              - "E2E (kyma-metrics)"
+              - "E2E (module-status-propagation)"
+              - "E2E (module-without-default-cr)"
+              - "E2E (non-blocking-deletion)"
+              - "E2E (purge-controller)"
+              - "E2E (purge-metrics)"
+              - "E2E (watcher-enqueue)"
+              - "Run 'make test'"
+              - "kyma (${{ matrix.cli-stability }}) alpha deploy -k config/${{ matrix.target }}"
```

Comment:
this one not working, should replaced with 
kyma (unstable) alpha deploy -k config/control-plane
kyma (unstable) alpha deploy -k config/default

Reply:
`lint` is missing